_topics/en/channel-jamming-attacks.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Channel jamming attacks
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Channel jamming attacks** are Denial of Service (DoS) attacks where
+  an attacker can prevent a series of channels up to 20 hops away from
+  being able to use part or all of their funds for a prolonged period of
+  time.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  An LN node can route a payment to itself across a path of 20 or more
+  hops. This creates two possible avenues for channel jamming attacks:
+
+  - *Liquidity jamming attack* (originally called the [loop attack][russell
+    loop] in 2015) is where an attacker with 1 BTC (for example) sends
+    it to themselves across 20 other channels but delays either settling
+    or rejecting the payment, temporarily locking up a total of 20 BTC
+    belonging to other users. After several hours of locking other
+    users’ money, the attacker can cancel the payment and receive a
+    complete refund on their fees, making the attack essentially free.
+
+  - *HTLC jamming attack* is where an attacker sends 483 small payments
+    ([HTLCs][topic HTLC]) through the series of 20 channels, where 483
+    is the maximum number of pending payments a channel may contain. In
+    this case, an attacker with two channels, each with 483 slots, can
+    jam over 10,000 honest HTLC slots—again without paying any fees.
+
+  ![Illustration of LN liquidity and HTLC jamming attacks](/img/posts/2020-12-ln-jamming-attacks.png)
+
+  A [variety][news120 upfront] of possible solutions have been
```

Comment:
Perhaps don't link to a specific mailing list post from the word 'variety'? (you link to this post in the optech_mentions below)